src/asset-management/EulerV2Manager.sol

```diff
@@ -305,17 +263,61 @@ contract AaveManager is IAssetManager, Owned(msg.sender), ReentrancyGuard {
                                 ADDITIONAL REWARDS
     //////////////////////////////////////////////////////////////////////////*/
 
-    function claimRewardForMarket(address aMarket, address aReward)
+    function claimRewards(
+        IDistributor aDistributor,
+        address[] calldata aUsers,
+        address[] calldata aTokens,
+        uint256[] calldata aAmounts,
+        bytes32[][] calldata aProofs
+    ) external onlyGuardianOrOwner {
+        aDistributor.claim(aUsers, aTokens, aAmounts, aProofs);
+
+        for (uint256 i = 0; i < aTokens.length; ++i) {
+            SafeTransferLib.safeTransfer(
+                aTokens[i],
+                msg.sender,
+                // the amounts specified in the argument might not be the actual amounts disimbursed by the distributor, due to the possibility of having done the claim previously
+                // thus it is necessary to use balanceOf to transfer the correct amount
```

Comment:
excessively long comment line, limit to 80 or 100

Reply:
fixed as well